kernel/interrupt.c

```diff
@@ -74,10 +74,11 @@ static void user_irq_queue_push(struct user_irq *uirq)
 
 static struct user_irq *user_irq_queue_pop(void)
 {
+	struct user_irq *uirq;
 	if (user_irq_queue_is_empty())
 		return NULL;
 
-	struct user_irq *uirq = user_irq_queue.head;
+	uirq = user_irq_queue.head;
```

Comment:
Will also do that, since I'm also moving sources (board, external, kernel, loader, platform, user) into a \src folder